src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -122,6 +122,7 @@ class ReportingConfigForm extends React.Component {
       }
     } else {
       // ...or create a new configuration
+      formData.append('enterprise_customer_id', this.props.enterpriseId);
```

Comment:
Can/Should we use `config.enterpriseCustomer.uuid` here like on https://github.com/openedx/frontend-app-admin-portal/pull/756/files#diff-d6c597698145cb29bdb34a6ead6524e9808691c30432db9f451bc697e99b9bd4R117

Reply:
`config` object is available only when we are updating an existing report config